purchase_order_type_dashboard/tests/test_purchase_order_type_dashboard.py

```diff
@@ -0,0 +1,65 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import time
+
+from odoo.tests import common, tagged
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+@tagged("post_install", "-at_install")
+class TestPurchaseOrderTypeDashboard(common.TransactionCase):
```

Comment:
According to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0 inheriting from BaseCommon is recommended.